### PR TITLE
Added html and css codes. Added all the sections.

### PR DESCRIPTION
**Pull request by daviidy**
Worked on all the requirements given on this project:

Built the heat map of the webpage
Used semantic HTML elements for the header, the main content, the form and the footer.
Use Flexbox, Grid and Float to align the elements.
Fixed linters errors

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+  font-size: 1.2rem;
+  text-decoration: underline;
+}
+
+.post a {
+  width: 100px;
+}
+
+.post i {
+  /* margin-top: 100px; */
```
Comment (arslanbisharat) [inline style.css:272]: Please don't use commented code in your files

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
**Hello Developers,**
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. A few changes are required. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**
- [x]  Please add links to your social accounts in readme file. 

chrome-capture (32)

- [x]  Please remove inline comments.


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great effort with the project!!

:100: loved that is almost identical to the original! Great work!

## Required changes
### [Mandatory] :red_circle: 

- [x] Issues and Licence link are broken in README.

- [x] All elements should be links and should be centered as the original website.
image

- [x] All elements should be links
image

- [x] The logo should be inside the navbar,

### [OPTIONAL]  :large_blue_circle: 

- [x] Improve project architecture by creating a directory that follows the next example:
```
/
  css/
    style.css
  img/
    ... some images...
  js/
    ... some scripts...
  index.html
```

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED 🥇 🌟 

Good job team, your project looks amazing! 👍 
You have met all the requirements and your project is approved! Congratulations! 💯 

Remember to fill up the completion form and you can move into the next project ⏩ 

Keep the hard work up, guys! 💪 

Good luck!  😃

**Pull request merged**